.github/workflows/Intel.yml

```diff
@@ -63,7 +63,7 @@ jobs:
         cd sp
         mkdir build
         cd build
-        cmake .. -DCMAKE_INSTALL_PREFIX=~/sp
+        cmake .. -DOPENMP=ON -DBUILD_SHARED_LIBS=ON -DCMAKE_INSTALL_PREFIX=~/sp
```

Comment:
I think we need to test both with and without shared libs. Let's get your PR merged and then I will circle back around and change the CI system to build with both shared and not shared.